CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.8.0 - 2024-09-20
+
+### Features
+
+- Apply const keyword to many methods ([#167](https://github.com/matrix-org/vodozemac/pull/167)).
+
+- [**breaking**] The `Account::sign()` method now accepts an `impl AsRef<[u8]>`
```

Comment:
```suggestion
- [**BREAKING**] The `Account::sign()` method now accepts an `impl AsRef<[u8]>`
```

Reply:
Did so, though we should update the cliff config if you prefer changelogs shouting at you.